src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        int index = new Random().nextInt(colors.length);
```

Comment:
Here, your random instance is created every time the method is called. random - is our dependency, let's initialize dependencies in the class fields.